tutorials/intro/plot_40_sensor_locations.py

```diff
@@ -96,6 +96,88 @@
 fig.gca().view_init(azim=70, elev=15)
 ten_twenty_montage.plot(kind='topomap', show_names=False)
 
+###############################################################################
+# .. _control-chan-projection:
+#
+# Controlling channel projection (MNE vs EEGLAB)
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Channel positions in 2d space are obtained by projecting their actual 3d
+# positions onto a sphere. Because ``'standard_1020'`` montage contains
+# realistic, not spherical, channel positions, we will use a different montage
+# to demonstrate controlling how channels are projected to 2d space.
```

Comment:
just a fly-by comment --> many users (including myself) will want to "fix" the plotting of `'standard_1020'` and not the biosemi montage. Can that be included in this brief example? :)

Reply:
This PR seems 98% ready to go (right @mmagnuski ?), in which case let's save that for https://github.com/mne-tools/mne-python/issues/7141 unless it's trivial to add here

Reply:
Unfortunatelly the `standard_1020` is not spherical, it contains realistic channel positions, I don't think it can be easily fixed. The easiest fix would be to distribute the spherical 1020 locations, which is discussed here: #7141 (as you know, for sure, cause you took part in the discussions :) ). Once this is done I'd happily change the tutorial to use the spherical 1020.

Reply:
alright, thanks - I guess I was hoping against all odds to find a ready-made solution here. Thanks for your PR!